Index: libx11-1.2.1/nls/compose.dir.pre
===================================================================
--- libx11-1.2.1.orig/nls/compose.dir.pre	2009-05-24 02:38:09.000000000 +1000
+++ libx11-1.2.1/nls/compose.dir.pre	2009-05-24 02:38:35.000000000 +1000
@@ -421,6 +421,7 @@
 en_US.UTF-8/Compose:		ko_KR.UTF-8
 en_US.UTF-8/Compose:		kw_GB.UTF-8
 en_US.UTF-8/Compose:		ky_KG.UTF-8
+en_US.UTF-8/Compose:		la_AU.UTF-8
 en_US.UTF-8/Compose:		lo_LA.UTF-8
 en_US.UTF-8/Compose:		lt_LT.UTF-8
 en_US.UTF-8/Compose:		lv_LV.UTF-8
Index: libx11-1.2.1/nls/locale.alias.pre
===================================================================
--- libx11-1.2.1.orig/nls/locale.alias.pre	2009-05-24 02:38:10.000000000 +1000
+++ libx11-1.2.1/nls/locale.alias.pre	2009-05-24 02:40:35.000000000 +1000
@@ -769,6 +769,7 @@
 ky_KG:							ky_KG.UTF-8
 ky_KG.utf8:						ky_KG.UTF-8
 ky_KG.UTF-8:					ky_KG.UTF-8
+la_AU.utf8:					la_AU.UTF-8
 lo:						lo_LA.MULELAO-1
 lo_LA:						lo_LA.MULELAO-1
 lo_LA.cp1133:					lo_LA.IBM-CP1133
Index: libx11-1.2.1/nls/locale.dir.pre
===================================================================
--- libx11-1.2.1.orig/nls/locale.dir.pre	2009-05-24 02:38:10.000000000 +1000
+++ libx11-1.2.1/nls/locale.dir.pre	2009-05-24 02:40:49.000000000 +1000
@@ -411,6 +411,7 @@
 ko_KR.UTF-8/XLC_LOCALE:			ko_KR.UTF-8
 en_US.UTF-8/XLC_LOCALE:			kw_GB.UTF-8
 en_US.UTF-8/XLC_LOCALE:			ky_KG.UTF-8
+en_US.UTF-8/XLC_LOCALE:			la_AU.UTF-8
 en_US.UTF-8/XLC_LOCALE:			lo_LA.UTF-8
 en_US.UTF-8/XLC_LOCALE:			lt_LT.UTF-8
 en_US.UTF-8/XLC_LOCALE:			lv_LV.UTF-8
